lib/rules/web/ocm_console/archived_cluster_detail.xyaml

```diff
@@ -13,7 +13,7 @@ archived_cluster_detail_page_loaded:
         - selector:
             xpath: //p[text()='The cluster has been archived and does not have any metrics data.']
         - selector:
-            xpath: //h2[@class='pf-c-title pf-m-3xl' and text()='Details']
+            xpath: //h2[@class='pf-c-title card-title' and text()='Details']
```

Comment:
Can we locate the element without class? If no, better to use `contains` rather than `=` for class due to the frequently change.

Reply:
Updated.